core/src/test/scala/akka/persistence/r2dbc/R2dbcSettingsSpec.scala

```diff
@@ -18,6 +18,21 @@ class R2dbcSettingsSpec extends AnyWordSpec with TestSuite with Matchers {
       settings.journalTableWithSchema shouldBe "s1.event_journal"
       settings.snapshotsTableWithSchema shouldBe "s1.snapshot"
       settings.durableStateTableWithSchema shouldBe "s1.durable_state"
+
+      // by default connection is configured with options
+      settings.connectionFactorySettings shouldBe a[ConnectionFactorySettings]
+      settings.connectionFactorySettings.urlOption should not be defined
+    }
+
+    "support connection settings build from url" in {
+      val config =
+        ConfigFactory
+          .parseString("akka.persistence.r2dbc.connection-factory.url=whatever-url")
+          .withFallback(ConfigFactory.load())
+
+      val settings = R2dbcSettings(config.getConfig("akka.persistence.r2dbc"))
+      settings.connectionFactorySettings shouldBe a[ConnectionFactorySettings]
+      settings.connectionFactorySettings.urlOption shouldBe defined
```

Comment:
The tests is really minimal. It only check the config object. 

We can move some DB tests to url based config eventually.